src/celeritas/em/distribution/UrbanMscHelper.hh

```diff
@@ -90,6 +87,7 @@ UrbanMscHelper::UrbanMscHelper(const UrbanMscRef&       shared,
     : inc_energy_(particle.energy())
     , physics_(physics)
     , dtrl_(shared.params.dtrl())
+    , range_(physics.dedx_range())
```

Comment:
Since the `MscScatter` is directly calling physics, I think you can delete this setter as well as the `range_` member data.

Reply:
Okay, I missed that part and drop the range_ member data.